misc.php

```diff
@@ -144,3 +144,10 @@ function wpcom_vip_wp_link_query_args( $query ) {
 }
 
 add_filter( 'wp_link_query_args', 'wpcom_vip_wp_link_query_args', 10, 1 );
+
+/**
+ * Stop Woocommerce from trying to create files on read-only filesystem
+ */
+if ( false !== VIP_GO_ENV ) {
```

Comment:
Should we do this everywhere to preserve production behaviour locally?

Reply:
Yeah, I have no strong argument for keeping this limitation. Someone can always remove the filter in their local environment, if it's causing them trouble.